webpack.config.js

```diff
@@ -17,10 +20,10 @@ module.exports = {
             loader: 'babel-loader',
             query: {
               presets: ["@babel/preset-react"]
```

Comment:
No `@babel/preset-env` as well?

Reply:
Huh. Nope? I guess I didn't notice that because I was running everything through modern browsers! I'll add that one in for completeness!